haiku-apps/heictranslator/heictranslator-0.2.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="HEIC Translator for Haiku"
+DESCRIPTION="A Haiku Translation Kit add-on to support HEIC image viewing"
+HOMEPAGE="https://github.com/dospuntos/HEICTranslator"
+COPYRIGHT="2025 Johan Wagenheim"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dospuntos/HEICTranslator/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="645951844c89f41a0710e35d954d834cbaedae478df81a830f1f298966fd29f0"
+SOURCE_DIR="HEICTranslator-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	heictranslator@secondaryArchSuffix = $portVersion
+	addon:HEICTranslator = $portVersion
+	"
+
```

Comment:
No empty line between PROVIDES and REQUIRES (see: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines)